templates/cronjob.yaml

```diff
@@ -0,0 +1,202 @@
+{{- if .Values.garbageCollect.enabled }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "docker-registry.fullname" . }}-garbage-collector
+  labels:
+    app: {{ template "docker-registry.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  schedule: {{ .Values.garbageCollect.schedule | quote }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
```

Comment:
Missing `spec.jobTemplate.spec.template.spec.serviceAccountName` (see https://github.com/twuni/docker-registry.helm/blob/main/templates/deployment.yaml#L37 for reference).

Reply:
Also missing `spec.jobTemplate.spec.template.spec.priorityClassName` (see https://github.com/twuni/docker-registry.helm/blob/main/templates/deployment.yaml#L44 for reference), although I'm not sure whether the intention carries over for this job as well as the registry pod(s).

Soft suggestion to add that in here, but if there's a good reason to omit it, non-blocking. I don't understand how `priorityClassName` is in use today, so can't make an informed judgment on that myself.